crates/services/txpool_v2/src/storage/mod.rs

```diff
@@ -102,4 +102,7 @@ pub trait Storage {
 
     /// Remove a transaction from the storage.
     fn remove_transaction(&mut self, index: Self::StorageIndex) -> Option<StorageData>;
+
+    /// Returns the number of transactions in the storage.
+    fn tx_count(&self) -> usize;
```

Comment:
You don't need to expose this finromation. `TxPool` already has `tx_id_to_storage_id` that contains this information.

Reply:
Updated in https://github.com/FuelLabs/fuel-core/pull/2385/commits/7737a144e5cb9417f36bbf1102143e6645878b04